geosys/ui/widgets/geosys_dockwidget.py

```diff
@@ -452,6 +455,27 @@ def restore_parameter_values_from_setting(self):
             value = setting(key, expected_type=int, qsettings=self.settings)
             value and form.setValue(value)
 
+    def set_parameter_values_as_default(self):
+        """Set parameter values to default values."""
+        for key, form in self.map_creation_parameters_settings.items():
+            print("KEY: " + str(key))
```

Comment:
@vermeulendivan we should remove this or use the plugin log if we want to log the key.

Reply:
@Samweli print statement has been removed